Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unrelased
+- [fixed] Made`FIRCLSContextInitialize` unblocking Main thread. (#13675, #13232)
```

Comment:
Suggestion: Improved startup time by putting some initialization steps on a background. Importantly, some crashes that happen during startup will not be caught.